### PR TITLE
Added cloud patterns

### PR DESCRIPTION
**Pull request by navveenb**
Added cloud patterns

### CODE REVIEW

**Comment by dubrie** [comment]
assigning to @dubrie for initial review

File: docs/catalog/cloud/adopt-serverless-architecture.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: cloud
+tags: 
+ - serverless
```
Comment (dubrie) [inline docs/catalog/cloud/adopt-serverless-architecture.md:7]: please add `cloud` to the tag list here

File: docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: cloud
+tags: 
+ - devops
```
Comment (dubrie) [inline docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md:7]: please include `cloud` in the list of tags

File: docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+# Leverage automation to start/stop instances and re-create non-production environments
+
+## Description
+
+Non-production workloads take up significant computing resources
+and adds to the carbon footprint of your application.
+
+Non-productions workloads, like development and test environments, need not be running all the time (like weekends,non-business hours, and extended non-utilization periods) and can be optimized through automation for efficient utilization.
```
Comment (dubrie) [inline docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md:19]: This seems to be duplicating an existing pattern in the catalog: https://patterns.greensoftware.foundation/catalog/cloud/minimizing-deployed-environments

Are there specific differences you want to call out in this proposal or can we update that pattern or mark this as a duplicate?

**Review by dubrie — CHANGES_REQUESTED** [review]
@navveenb a few changes and considerations for the initial review

File: docs/catalog/cloud/adopt-serverless-architecture.md (hunk 1 above)
Comment (navveenb) [reply docs/catalog/cloud/adopt-serverless-architecture.md:7]: Added @dubrie . Thanks

File: docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md (hunk 2 above)
Comment (navveenb) [reply docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md:7]: Added tag

File: docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md (hunk 3 above)
Comment (dubrie) [reply docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md:20]: Great! I'll mark this PR as closed then and wait for the new one with the updates to the existing pattern. Thanks!

**Comment by dubrie** [comment]
Closing as duplicate pattern

**Pull request closed without merging**